helpdesk/tests/test_get_email.py

```diff
@@ -20,35 +20,114 @@
     # Python < 3.3
     import mock
 
+unrouted_socks_server = "127.0.0.1"
```

Comment:
For that sense of unrouted I think the safest address would be an invalid one, e.g. 0.1.1.1. You're guaranteed no traffic is going anywhere that way...